docs/pc-intel-x86_32/build/build-core.rst

```diff
@@ -29,4 +29,6 @@ Copy this to the target machine's ``/boot/``
    below
 #. ``debug_serial`` to enable debug output via UART; you must 
    additionally specify the debug port number via ``debug_port`` (e.g., 0x3f8 for COM1)
-
+#. ``debug_memory`` to enable debug output to memory. Currently this configuration 
+   works only when debug_serial is enabled. This is because one has to find the 
+   base address of the log memory in the serial output.

```

Comment:
I don't believe we need to have serial debug enabled for debug memory to work. We could add a couple of lines here to use `readelf` on the uxmhf elf binary to locate `_log_mem` variable for DCI debugging. Please see comment on _log_mem below as well.